### DISCUSSION TITLE
Cannot figure out the ambient temperature parameter value

### DISCUSSION THREAD

**Discussion by Guri080** [pybamm-team/PyBaMM#4566] [Q&A]
I am mostly interested in the discharge capacity and cycle number retrieved from the summary variables. I don't see many issues when I set the ambient temperature parameter value to room temperature but when I set it to something really low like 283.15 K (10 degrees C) it either produces a wrong graph or does not change much. I was thinking of implementing the temperature dependency, but I can't locate where the summary variables are populated and where the parameter values play a role. I would greatly appreciate any advice.

**Comment by rtimms** [comment]
Which model and parameter values are you using? Many parameters can depend on temperature, but the temperature dependence isn't included in all parameter sets.

**Reply by Guri080** [reply]
Hello rtimms,
Thank you for your reply. I am using the SPM model and chen2020 parameter values. I do update the initial concentration in positive and negative electrodes, SEI Kinetic constant, nominal cell voltage, and lower and upper voltage cut-off parameters

**Reply by Guri080** [reply]
Hey rtimms,

Just a quick follow-up regarding our conversation
